Main.cpp

```diff
@@ -140,20 +140,22 @@ enum StarSize {
 
 struct Star {
     // Bitmap representation
-    BITMAP *m_pBitmap;
+    BITMAP *m_pBitmap = nullptr;
```

Comment:
You can kill off all these inline comments here about Bitmap representation and stuff.
Also, I know this is on me with the styleguide but we're getting rid of the _p in member variables cause VS makes it clear what type a variable is. So m_pBitmap should become m_Bitmap.